smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/swiftmodules/SmithyTestUtilTypes.kt

```diff
@@ -1,17 +1,19 @@
 package software.amazon.smithy.swift.codegen.swiftmodules
 
 import software.amazon.smithy.codegen.core.Symbol
+import software.amazon.smithy.swift.codegen.SwiftDeclaration
 import software.amazon.smithy.swift.codegen.SwiftDependency
-import software.amazon.smithy.swift.codegen.model.buildSymbol
 
 object SmithyTestUtilTypes {
     val TestInitializer = runtimeSymbol("TestInitializer")
     val TestBaseError = runtimeSymbol("TestBaseError")
-    val SelectNoAuthScheme = runtimeSymbol("SelectNoAuthScheme")
+    val dummyIdentityResolver = runtimeSymbol("dummyIdentityResolver", SwiftDeclaration.FUNC)
```

Comment:
Adds a symbol for `dummyIdentityResolver` and updates the helper function used below to correct symbols.